odoo/cli/command.py

```diff
@@ -23,46 +27,83 @@ def __init_subclass__(cls):
 
 Use '{odoo_bin} <command> --help' for individual command help."""
 
+
 class Help(Command):
     """ Display the list of available commands """
     def run(self, args):
-        padding = max([len(cmd) for cmd in commands]) + 2
+        populate_internal_commands()
+        populate_addons_commands()
+        padding = max(len(cmd) for cmd in commands) + 2
         command_list = "\n    ".join([
             "    {}{}".format(name.ljust(padding), (command.__doc__ or "").strip())
-            for name, command in sorted(commands.items())
+            for name in sorted(commands)
+            if (command := find_command(name))
         ])
-        print(ODOO_HELP.format(  # pylint: disable=bad-builtin
+        print(ODOO_HELP.format(  # pylint: disable=bad-builtin  # noqa: T201
             odoo_bin=Path(sys.argv[0]).name,
             command_list=command_list
         ))
 
+
+def populate_internal_commands():
```

Comment:
The term "populate" does not seem to relate to `commands`, which is the dict of _loaded_ commands.  I suggest to call the function(s) `load_internal_commands()`, etc.